src/screens/AccountList.js

```diff
@@ -138,16 +112,16 @@ class AccountListView extends React.PureComponent {
             this.list = list;
           }}
           style={styles.content}
-          data={accounts}
-          keyExtractor={account => accountId(account)}
+          data={[...accounts.entries()]}
```

Comment:
why not just `data={accounts.entries()}`?

Reply:
Your suggestion gives:
> Error: Invariant violation: tried to get frame for out of range index NaN

Probably because `entries()` from a map returns a map iterator object that Flatlist can't handle. (cf: [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/entries#Syntax))